tools/serverpod_cli/lib/src/generator/dart/library_generators/entities_library_generator.dart

```diff
@@ -1369,6 +1402,330 @@ class SerializableEntityLibraryGenerator {
     }));
   }
 
+  Class? _buildEntityRepositoryClass(
+    String className,
+    List<SerializableEntityFieldDefinition> fields,
+    ClassDefinition classDefinition,
+  ) {
+    // TODO: remove when this class supports db operations
+    if (!(_hasAttachOperations(fields) || _hasDetachOperations(fields))) {
+      return null;
+    }
```

Comment:
Suggestion: Move to call site.

Since this determines if the class is built or not I think it belongs to the abstraction layer above. Could you move this out to the call site to make it clear if we build the class or not at that place?